app/jobs/api/issues_unlabel_issue_job.rb

```diff
@@ -2,9 +2,17 @@ module Api
   class IssuesUnlabelIssueJob < IssueEventJob
     timestamp ->(params) { params[:issue]['updated_at'] }
     action "issue_unlabeled"
+    cache_key ->(message) {
+      label_name = message[:payload][:label] ? message[:payload][:label]['name'] : ""
+
+      "#{message[:meta][:action]}.#{message[:meta][:user]["login"]}.#{message[:meta][:identifier]}.#{message[:meta][:timestamp]}.#{ label_name }"
```

Comment:
> Leverage a class method so that we don't screw this up

Little confused, `cache_key` is a class method thats taking a lambda as its argument.